src/components/forms/Input.js

```diff
@@ -58,8 +62,14 @@ export default function Input({
   }
 
   return (
-    <div className="col-span-full">
-      <label htmlFor={id} className="block text-sm font-medium leading-6">
+    <div className={classNames("col-span-full ", className)}>
```

Comment:
Extra space not needed here between strings, the function will do it for us
```suggestion
    <div className={classNames("col-span-full", className)}>
```